src/quickdesktopwallpaper_linux.cpp

```diff
@@ -22,9 +22,13 @@
  * SOFTWARE.
  */
 
+#define QT_NO_KEYWORDS
```

Comment:
Maybe this should be added in the CMakeLists file, and you may also need `#undef signal` in this file